test/libsolidity/SolidityOptimizer.cpp

```diff
@@ -84,9 +85,20 @@ class OptimizerTestFramework: public ExecutionFramework
 
 	AssemblyItems getCSE(AssemblyItems const& _input)
 	{
+		// add dummy locations to each item so that we can check that they are not deleted
+		AssemblyItems input = _input;
+		for (AssemblyItem& item: input)
+			item.setLocation(SourceLocation(1, 3, make_shared<string>("")));
+
 		eth::CommonSubexpressionEliminator cse;
-		BOOST_REQUIRE(cse.feedItems(_input.begin(), _input.end()) == _input.end());
-		return cse.getOptimizedItems();
+		BOOST_REQUIRE(cse.feedItems(input.begin(), input.end()) == input.end());
+		AssemblyItems output = cse.getOptimizedItems();
+
+		for (AssemblyItem const& item: output)
+		{
```

Comment:
Yes, for compiling on windows, the macro seems to not expand to a single statement.